beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -713,12 +615,12 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         &self,
         start_slot: Slot,
     ) -> Result<impl Iterator<Item = Result<(Hash256, Slot), Error>> + '_, Error> {
-        let local_head = self.head()?;
+        let local_head = self.head_snapshot();
 
         let iter = self.store.forwards_state_roots_iterator(
             start_slot,
             local_head.beacon_state_root(),
```

Comment:
Unsure if this matters in this case, but the old version of this function called `head()` which used to use `CloneConfig::committee_caches_only()` instead of `CloneConfig::none()`

Reply:
Great pickup, very impressive! @michaelsproul and I discussed this at the time and decided that the committee caches weren't necessary here.